### DISCUSSION TITLE
LongPressCommand

### DISCUSSION THREAD

**Discussion by fredericoregateiro** [CommunityToolkit/Maui#567] [Q&A] [answered]
Hello,
I'm migrating a xamarin app that uses the Xamarin.CommunityToolkit and i cannot find the `xct:TouchEffect.LongPressCommand` equivalent in the Maui CommunityToolkit, 

Is it missing here? 
Thanks,

Frederico

**Comment by bijington** [comment, marked as answer]
This feature is planned however it has not been added yet. You can follow its progress here: https://github.com/CommunityToolkit/Maui/issues/86

It is also worth noting that it won't be called `TouchEffect` in this toolkit because we are moving away from the legacy `Effects` API. It will likely become `TouchBehavior`.

Of course if you would be willing to contribute the functionality it would be a huge help to us and the community 🙂

**Comment by CliffAgius** [comment]
Closed as answered
